.github/ISSUE_TEMPLATE.md

```diff
@@ -1,11 +1,9 @@
-Describe the task here...
+Describe the task here and link or add dependencies to any applicable issues...
 
 ### This task is done when...
  - [ ] criteria 1
  - [ ] criteria 2
  
-*Don't forget to add the LABEL and MILESTONE before submitting.
+*Add a LABEL (design, dev, compliance, BUG, etc) before submitting.
 
-*If the issue is for the CURRENT SPRINT, add it to the PROJECT.
-
-*If the issue is a BUG, or FUTURE ENHANCEMENT, do not add it to the PROJECT
+*After submitting, the issue will go into the _unprioritized pipeline_ for PRODUCT to prioritize.
```

Comment:
We should include some clarification here (since this has come up on our retro/sprint discussions):
1) Issues needed to complete active, prioritized work in the sprint may be added to the "This Sprint" pipeline. Examples: Breaking up existing issues into smaller issues.
2) Issues related to new work, or if you are unsure where to place the issue, should be added to the "Unprioritized" pipeline.

Reply:
You are correct, but is this content for the issue template? I was thinking that this info about where issues go and when, should actually go in our "How we work" wiki page. Does that make sense?

Reply:
I see what you are thinking... hmm... If that's the case we should just remove that line OR we could update the old language (I don't think people are always reading the "How we work" section -- some of our conversations in sprint planning/standup have been to remind people "How we work.")
*If the issue is for the CURRENT SPRINT, add it to the "This Sprint" pipeline. 
*If the issue is a BUG, or FUTURE ENHANCEMENT, the issue will go into the _unprioritized pipeline_ for PRODUCT to prioritize.

Reply:
Changes look good.